.github/workflows/ci-pipeline-arm64.yml

```diff
@@ -98,7 +98,8 @@ jobs:
         conf: ['', '--enable-abi-compat', '--enable-deprecated --enable-debug=full',
                '--enable-dpdk-zero-copy --disable-static-applications',
                '--disable-host-optimization', '--disable-host-optimization --enable-abi-compat',
-               '--without-openssl --without-pcap']
+               '--without-openssl --without-pcap', 'CFLAGS="-O2 -march=armv8.2-a"',
+               'CFLAGS="-O2 -march=armv8-a+lse"']
```

Comment:
It looks like the additional quotation marks are not handled correctly, which causes the CI failures. We may need to do some trial and error to figure out a syntax that works.

Reply:
I will push another version with some changes and we can see if it works. Unfortunately, there is no way for me to run the arm64 jobs in my GitHub account so I'll keep submitting new versions until we figure it out.